test/fixtures/example_healthcert_invalid_file.json

```diff
@@ -0,0 +1,180 @@
+{
```

Comment:
Do any of the notification-related tests depend on these fixtures?

Reply:
Only the `example_healthcert_with_nric_wrapped.json` fixture is used by [src/models/notifyRecipient/notifyRecipient.test.ts](https://github.com/Notarise-gov-sg/sns-notify-recipients/blob/905162fab988b314baba126a2fbd3447d6a6c935/src/models/notifyRecipient/notifyRecipient.test.ts).

Shall I delete the rest of the fixtures?

Reply:
Yup, please go ahead, so that we don't have files that aren't needed